app/views/spree/products/_qna.html.erb

```diff
@@ -1,17 +1,17 @@
 <% if @product.product_questions.answered.visible.present? %>
   <div class="row">
-    <h6 class="product-section-title"><%= t(:questions_and_answers) %></h6>
+    <h4 class="product-section-title w-100  font-weight-bold"><%= t(:questions_and_answers) %></h4>
```

Comment:
seems like two spaces exist in ` w-100  font-weight-bold`, can you remove one space?